example_configs/cassandra-datastax-driver.yml

```diff
@@ -0,0 +1,35 @@
+---
+lowercaseOutputName: true
+rules:
+# These metrics was re-added to a dropwizard-context, so they were partly renamed.
+# In jmx they will appear as:
+# $cluster_name$-metrics{name=requests}[]Count: 48
```

Comment:
I'm confused by this and the rules you have below - is the cluster name there or not?